install_config/cluster_metrics.adoc

```diff
@@ -306,15 +340,15 @@ ifdef::openshift-enterprise[]
 ====
 endif::[]
 
-
-You will need to save your completed file with the file name *_metrics.yaml_*.
+Copy this file to another directory with the file name
+*_metrics-deployer.yaml_*.
```

Comment:
I wasn't considering the case where people would be editing the file, for that scenario then it does make sense to copy it across. Thanks for the explanation.